frontend/tests/e2e/pageObjects/studentAdmin/institute/schoolsPage.js

```diff
@@ -0,0 +1,49 @@
+import { Selector, t } from 'testcafe'
+const log = require('npmlog')
+
+class SchoolsPage {
+  constructor() {
+    //search box
+    this.schoolNameTextField = Selector('#name-text-field');
+    this.schoolStatusSelectField = Selector('#status-select-field');
+    this.schoolSearchButton = Selector('#user-clear-button');
+
+    //search results
+    this.schoolSearchResultsText = Selector('tr td');
+  }
+
+  async clickSearchButton() {
+    await t.click(this.schoolSearchButton());
+    log.info("search button clicked");
+  }
+
+  async setName(text) {
+    await t.typeText(this.schoolNameTextField, text).wait(10000);
+    log.info(`School name:: ${text} entered`);
```

Comment:
hmm does this wait for 10 seconds after the text is input? Seems a bit long. Pretty sure you can get away without the waits.